packages/grpc-js/test/common.ts

```diff
@@ -16,7 +16,7 @@
  */
 
 import * as loader from '@grpc/proto-loader';
-import * as assert from 'assert';
+import * as assert2 from './assert2';
```

Comment:
@murgatroid99 this is one material change I made (although it's really just a simple refactor).

It just removes the exporting of a namespace.